docs/flex/interactive/tools/cypher_shell.md

```diff
@@ -0,0 +1,20 @@
+# Cypher Shell
+
+GraphScope interactive is compatible with the existing neo4j cypher ecosystem. Users can use Cypher Shell as a client to connect to the database service and submit queries for execution.
+
+## Downloading
+We currenly only support Cypher Shell version 4.4.22. You can download from the [official release](https://dist.neo4j.org/cypher-shell/cypher-shell-4.4.22.zip?_ga=2.82138006.1038582604.1690942647-1290743114.1671172421).
```

Comment:
remove the tracing code? use this instead. 
https://dist.neo4j.org/cypher-shell/cypher-shell-4.4.22.zip